docs/pages/includes/install-linux.mdx

```diff
@@ -62,6 +62,17 @@ Use the appropriate commands for your environment to install your package.
   # $ sudo dnf install teleport
   ```
 
+  <Admonition type="warning" title="Amazon Linux 2023 Installations">
+  Amazon Linux 2023 installations should utilize the Tarball installation or
```

Comment:
"install" appears 4 times in this sentence.
```suggestion
  Amazon Linux 2023 users should utilize the tarball or RPM file to install Teleport.
```

Reply:
thanks @zmb3 , applied that and updated the following sentence.